WalletWasabi.Fluent/Behaviors/RestartLinuxRendererBehavior.cs

```diff
@@ -0,0 +1,31 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Controls;
+using Avalonia.Xaml.Interactivity;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+// TODO: This is a workaround for this Avalonia issue (https://github.com/AvaloniaUI/Avalonia/issues/11850)
+// Please, remove this Behavior and it's unique usage after it's been fixed.
+public class RestartRendererBehavior : DisposingBehavior<Window>
```

Comment:
To match the file name and the class name.

```suggestion
public class RestartLinuxRendererBehavior : DisposingBehavior<Window>
```

Reply:
That name isn't exact either :)

The "renderer" refers to the abstraction itself, not the implementation. The problem is that the renderer is halted IN linux. So, a better name would be "RestartRendererInLinuxBehavior".